src/netaddress.h

```diff
@@ -140,7 +140,6 @@ class CService : public CNetAddr
         CService(const struct in_addr& ipv4Addr, unsigned short port);
         CService(const struct sockaddr_in& addr);
         void Init();
-        void SetPort(unsigned short portIn);
```

Comment:
Still, unused code is just dead weight and confusing. In this case, it implies that something external to the class is setting the port, but nothing is.